data/component-spelling.txt

```diff
@@ -23,6 +23,7 @@ cohort
 comment
 competency
 completion
+configdata
```

Comment:
This file is an automatically generated list of subsystems and plugintypes (it will eventually also include the standard moodle plugin list too).
Manual changes to this will get automatically removed on Sunday.

You probably want to put these in project-words.txt instead.

Reply:
Thanks for pointing me in the right direction here. I have moved these to project-words.txt.